h/browser/chrome/lib/fetch.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * Performs a network fetch for JSON data and returns a Promise
+ * for the result.
+ */
+function fetchJSON(url) {
```

Comment:
`fetch()` was made stable in Chrome 42 (April 2015). I figured that's still a little too recent. We could use the 'isomorphic-fetch' polyfill (http://caniuse.com/#feat=fetch) and then just remove that at some point in future.